openapi/components/parameters/orderExpand.yaml

```
@@ -0,0 +1,20 @@
+name: expand
+in: query
+description: |-
+  Expand a response to receive a full related object in the `_embedded` path.
+
+  To expand multiple objects, use a comma-separated list.
+  Example: `expand=customer,initialInvoice`.
+
+  Available arguments are:
+
+    - `customer`
+    - `initialInvoice`
+    - `recentInvoice`
+    - `upcomingInvoice`
+    - `paymentInstrument`
+    - `website`
+
+  For more information, see [Expand to include embedded objects](../../overview/#section/Expand-to-include-embedded-objects).
```

nitpick: I think we need to use an absolute link here
```suggestion
  For more information, see [Expand to include embedded objects](https://www.rebilly.com/catalog/all/#section/Expand-to-include-embedded-objects).
```